wp-content/themes/pawar2018/page-fullwidth.php

```diff
@@ -18,6 +18,58 @@
           <?php the_content(); ?>
         </div>
       </div>
+      <div class="row align-center">
+          <?php if (get_field('has_pillars')) : ?>
+            <?php
+              $args = array(
+              'post_type' => 'pillars',
+              'orderby' => 'meta_value',
+              'order' => 'ASC') ?>
+            <?php $loop = new WP_Query( $args ); ?>
+            <?php while ( $loop->have_posts() ) : $loop->the_post(); ?>
+              <div class="feature-issue small-12 medium-10 large-5">
+                  <img src="<?php echo get_field('pillar_icon') ?>" class="issue-icon" alt="<?php echo get_field('pillar_icon_alt') ?>">
+                  <div class="issue-summary">
+                    <h6><?php the_title();?></h6>
+                    <p><?php the_content() ?></p>
+                  </div>
+              </div>
+              <?php
+              // clean up after the query and pagination
+              wp_reset_postdata();
+              ?>
+            <?php endwhile; ?>
+          <?php endif; ?>
+        </div>
+    </div>
+  </section>
+      <?php if (get_field('has_issues')) : ?>
+      <section class="issue-accordion">
+        <div class="row align-center">
+          <div class="small-12 columns">
+            <ul class="accordion">
+            <?php
+              $args = array(
+              'post_type' => 'issues',
+              'orderby' => 'title',
+              'order' => 'ASC') ?>
+            <?php $loop = new WP_Query( $args ); ?>
+            <?php while ( $loop->have_posts() ) : $loop->the_post(); ?>
+              <li class="accordion-item">
+                <a href="#<?php global $post; $post_slug=$post->post_name; echo $post_slug; ?>" name="<?php global $post; $post_slug=$post->post_name; echo $post_slug; ?>" class="accordion-title"><?php the_title();?></a>
```

Comment:
Two things here:

1) Need to remove the `preventDefault` from `app.js` when clicking these links to get the benefits of the anchor link.

2) Should clean up the PHP to be a bit more DRY. E.g., no need to globalize `$post` or assign `$post_slug` multiple times. Could do that once right after the opening `while` and just perform the `echo` in each spot.